markdown/developers/sync.md

```diff
@@ -358,58 +361,58 @@ That's it, you're done! **Congratulations!**
 
 # Fixing a broken TEMPLATE branch
 
-Sometimes mistakes can occur during resolving of conflicts during a template merge, typically via the GitHub interface, where the changes during the conflict
-resolution is committed to `TEMPLATE` rather than the `dev` branch itself.
+If merge conflicts are resolved via the GitHub interface instead of after pulling changes to a fork as described above, the commit history from the `dev` branch will be merged into `TEMPLATE`.
 This leads to complex problems in later `TEMPLATE` merges as the later updated `TEMPLATE` branch removes all the pipeline-specific files that were accidentality included in problematic merge, resulting in many (in some cases >100!) of files to resolve conflicts in.
 
-If during one of the automated syncs you see you have an usually high number of changed files you can check whether `dev` was accidentally merged into `TEMPLATE` by looking at your repository's GitHub Network Graph under the 'Insights' tab. You can look through the pipeline history to see if there exists a commit described as 'Merge branch `dev` into `TEMPLATE`'.
+If during one of the automated syncs you see you have an usually high number of changed files you can check whether `dev` was accidentally merged into `TEMPLATE` by looking at how many commits the `TEMPLATE` branch has (should be in the range of 5-15ish, depending on how many template updates your pipeline has had). You can also look at your repository's GitHub Network Graph under the _"Insights"_ tab or even look through the `TEMPLATE` branch commit history to see if there is a commit described as 'Merge branch `dev` into `TEMPLATE`'.
 
 If so, the easiest solution is to start your `TEMPLATE` branch from scratch.
 
-To do this, clone your repository to your local machine, ensuring you have pulled both `TEMPLATE` and `dev` branches.
+* Clone the main nf-core pipeline repository to your local machine (not your development fork)
 
-Next, retrieve the commit hash when the original cookie-cutter template was used to generate pipeline i.e. with `nf-core create`.
-Assuming you originally started with the nf-core template, you can simply look at your git log from within your repository:
+  ```bash
+  git clone <git URL>
+  cd <pipeline>
+  ```
 
-```bash
-git log --reverse
-```
+* Next, retrieve the commit hash when the original nf-core template was used to generate pipeline i.e. with `nf-core create`.
+  * Assuming you originally started with the nf-core template, you can simply look at your git log from within your repository:
 
-The first commit will then typically represent the original template.
+    ```bash
+    git log --reverse
```

Comment:
- On which branch should we do that? (I'm assuming `TEMPLATE`

Reply:
Shouldn't matter here actually as all branches should have the same root commit. But it will matter when we reset in the later command!

Reply:
```suggestion
    git checkout TEMPLATE
    git log --reverse
```